sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/src/scenarios/inc/scope_guard.hpp

```diff
@@ -0,0 +1,184 @@
+/*
+ *  Created on: 13/02/2018
+ *      Author: ricab
+ *
+ * See README.md for documentation of this header's public interface.
```

Comment:
Is this MSFT work or 3rd party?  What's the license for it?

Reply:
3rd party. The license is that it's uncopyrighted material.  See [this](https://github.com/ricab/scope_guard/blob/main/LICENSE).

It's also tracked in the cgmanifest.json file.